packages/app/src/App.tsx

```diff
@@ -33,7 +33,7 @@ const app = createApp({
     SignInPage: props => (
       <SignInPage
         {...props}
-        providers={['guest', 'google', 'custom', 'okta', 'gitlab']}
+        providers={['guest', 'google', 'custom', 'okta', 'gitlab', 'github']}
```

Comment:
Addressing the first question about conflicts in this list here. It's not intended that the SignInPage is the actual page you'd use in a real app deployment inside an organization. It doesn't really make sense to allow a bunch of different providers inside an org, where you'd typically have a single one selected. This page is acting more as a utility for development, demo setup, example of using different providers, and a quick way to get started.

We'll probably look at doing a SignInPage with a single login option at some point.